lib/diamonds.ts

```diff
@@ -0,0 +1,327 @@
+import {
+  IDiamondReadable,
+  IDiamondWritable,
+} from '@solidstate/typechain-types';
+import { Contract, constants } from 'ethers';
+
+const Table = require('cli-table3');
```

Comment:
I'm conflicted about whether SolidState should include pretty print functions.  CLI features should be included in dedicated CLI modules, such as Hardhat plugins.  But such a plugin needs planning and might not cover all use cases for pretty printing.

Reply:
Admittedly I wrote this function as an experiment in visualizing the diamond facets. I agree though, this could very easily be a HH plugin.

Reply:
Let's keep it here for now but not export it as part of the API.  We'll move it to a plugin later, and for now focus on the other functions.

Reply:
https://github.com/solidstate-network/solidstate-solidity/pull/158/commits/b2eda5732ea11020a502eea1572d8f5d472de046

Reply:
https://github.com/solidstate-network/solidstate-solidity/pull/158/commits/f9a6c2c2c162b15dd1c97c4f5fd9b01eba971469